cv32e40s/env/corev-dv/cv32e40s_csr_template.yaml

```diff
@@ -72,88 +72,6 @@
 #      - type: ...
 #      - ...
 
-# User CSRs not implemented for first release of CV32E40S
```

Comment:
This file is outdated.
The new location, for a single source of truth, will be in the RTL repo.
Should I just `git rm` this one?

Reply:
Yes, if the file is truly out-dated then pls delete it.  No need to `git rm` as that will delete it from all previous hashes.  It should be sufficient to simply delete the file as part of the commit - this will remove it from clones from future hashes.